automatic/graphviz/graphviz.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>graphviz</id>
+    <version>0.0</version>

```

Comment:
Should this be 2.44.1? Or is this field ignored?

Reply:
Auto filled by AU upon being ran to update.